spec/models/reply_spec.rb

```diff
@@ -187,4 +187,22 @@
       include_examples 'has_edit_audits', method
     end
   end
+
+  describe "#reply_order" do
+    let(:post) { create(:post) }
+
+    it "puts replies in the correct order" do
+      first_reply = create(:reply, post: post)
+      second_reply = create(:reply, post: post)
+      third_reply = create(:reply, post: post)
+      expect(post.replies.ordered).to eq([first_reply, second_reply, third_reply])
+    end
+
+    it "puts replies in the correct order even if they are created in the wrong order" do
+      first_reply = create(:reply, post: post)
+      second_reply = Timecop.freeze(first_reply.created_at - 5.seconds) { create(:reply, post: post) }
+      third_reply = Timecop.freeze(first_reply.created_at - 3.seconds) { create(:reply, post: post) }
+      expect(post.replies.ordered).to eq([first_reply, second_reply, third_reply])
```

Comment:
Maybe compare this to `post.replies.order(:created_at)` to show the difference (and ensure it's all working as expected)?